_posts/2022-05-04-#24584.md

```diff
@@ -10,15 +10,23 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* One technique used for determining the payment address and amount is the ["Payment to different script type"](https://en.bitcoin.it/wiki/Privacy#Sending_to_a_different_script_type) heuristic.
```

Comment:
Maybe start by summarizing what information we are trying to keep private and why, then describe the technique someone might use to discover that information.